packages/core/docs/angular.stories.mdx

```diff
@@ -3,6 +3,8 @@ import { html, LitElement } from 'lit-element';
 
 <Meta title="Documentation/Angular" />
 
+<Story id="stories-clarity--theme-switcher" inline={true} />
+
```

Comment:
This is a placeholder story for the storybook pages that are plain text content. The decorator theme switcher only runs if there is at least one story on the page.